docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -44,13 +46,18 @@ The **Variables** panel tracks the data collected. Global variables are shown by
 
 Play executes all logic of the process and its linked files, such as FEEL, forms, DMN tables, and outbound Connectors.
 
-However, actions in Play cannot be triggered by any external system, such as external user interfaces, job workers, message systems, or inbound Connectors.
+Actions in Play can be initiated through Operate, Tasklist, or external APIs. For example, you can complete a user task via Tasklist, finish a service task using an external job worker, or cancel/modify your instance through Operate, with all changes reflected in Play.
+
+In SaaS, view your process instance in Operate by selecting the **Process Instance Key** in the header.
```

Comment:
> is this build up to date?
no, it isn't, since Web Modeler release didn't happen yet. you can use this image for web-modeler-webapp:

registry.camunda.cloud/web-modeler/modeler-webapp-self-managed:ci-3fe9661e2de3bb1fdc595d25b55c448422d66d2f (you need to update docker-compose/camunda-8.6/docker-compose.yaml file) see screenshot below:

<img width="943" alt="Screenshot 2024-08-28 at 22 13 58" src="https://github.com/user-attachments/assets/1a9c1e10-1dae-4c40-9bbe-6de3ee118e1d">

Reply:
but it will be available with alpha5 release.

Reply:
Thank you! You can disregard any of my questions about functionality limited to that build, since I wasn't able to test it at the time. 👍